akka-http-tests/src/multi-jvm/scala/akka/http/AkkaHttpServerLatencyMultiNodeSpec.scala

```diff
@@ -176,7 +176,7 @@ class AkkaHttpServerLatencyMultiNodeSpec extends MultiNodeSpec(AkkaHttpServerLat
         enterBarrier("load-gen-ready")
 
         runOn(server) {
-          val (_, port) = SocketUtil.temporaryServerHostnameAndPort()
+          val port = 0
```

Comment:
this means we'll do `setServerPort(0)` later - I'd say that probably would make tests that try to use that port fail? Perhaps it'd be OK to wait for the binding to complete here and pass in the actual port.

Reply:
Are you suggesting something like this: `setServerPort(_binding.map(_.localAddress.getPort).getOrElse(port))`?

Reply:
No, since the `getOrElse(port)` would still produce an unusable port. I'm not sure we would get away with `_binding.futureValue.localAddress.getPort` here?

Reply:
Changed to:

```         
val binding = Http().newServerAt("0.0.0.0", port).bind(routes).futureValue
_binding = Some(binding)
setServerPort(binding.localAddress.getPort)
```